### PR TITLE
Change name of Next-Gen instance (from Private to Next-Gen)

### PR DESCRIPTION
**Pull request by akharche**

### CODE REVIEW

File: bb/master/config.py [hunk 1]
```diff
@@ -208,22 +208,22 @@ class Mode(Enum):
...
+elif CURRENT_MODE == Mode.PRODUCTION_MODE_NEXT_GEN:
+    BUILDBOT_TITLE = "MediaSDK Next-Gen"
     WORKERS = {"build": {"b-50-41": {},
                          "b-50-61": {}},
                "ubuntu": {"b-999-999": {}},
                "centos_test": {"t-999-999": {}}}
     GITHUB_OWNERS_REPO = msdk_secrets.EMBEDDED_REPO
     BUILDBOT_URL = msdk_secrets.BUILDBOT_URL
-    MASTER_PRODUCT_TYPE = "embedded_private"
+    MASTER_PRODUCT_TYPE = "next_gen"
```
Comment (adydychk) [inline bb/master/config.py]: If you change this variable need to change product-type in `build_scripts/build_runner.py`

File: bb/master/config.py [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
 class Mode(Enum):
     PRODUCTION_MODE = "production_mode"
-    PRODUCTION_MODE_PRIVATE = "production_mode_private"
+    PRODUCTION_MODE_NEXT_GEN = "PRODUCTION_MODE_NEXT_GEN"
```
Comment (adydychk) [inline bb/master/config.py]: Make this string low case (lines are in different style - bad)

**Comment by adydychk** [comment]
Code is OK.
It needs to deploy it.

File: build_scripts/build_runner.py [hunk 3]
```diff
@@ -894,7 +894,7 @@ def main():
     parser.add_argument('-p', "--product-type", default='linux',
                         choices=['linux', 'embedded', 'open_source', 'windows',
                                  'windows_sw_lib', 'windows_hw_lib', 'windows_tools', 'windows_mfts',
-                                 'windows_uwp', 'api_latest', 'embedded_private', 'android',
+                                 'windows_uwp', 'api_latest', 'next_gen', 'android',
```
Comment (azhogov) [inline build_scripts/build_runner.py]: should be linux_next_gen

File: bb/master/config.py (hunk 1 above)
Comment (adydychk) [inline bb/master/config.py]: I see different name in build_runner.py - `linux_next_gen`. Fix it.

Comment (adydychk) [reply bb/master/config.py]: Fix it.

**Pull request merged**